cg/meta/demultiplex/status_db_storage_functions.py

```diff
@@ -55,6 +56,19 @@ def store_flow_cell_data_in_status_db(
     store.session.commit()
 
 
+def check_if_samples_have_files(flow_cell_path: Path, sample_ids: List[str]) -> None:
+    """Check if all samples have already fastq files in the demultiplex directory.
```

Comment:
Check if all samples have fastq files in the flow cell directory maybe (?)

Reply:
I think is in the demultiplex directory right? because we are finishing the flow cell